libkbfs/interfaces.go

```diff
@@ -65,7 +65,8 @@ type diskBlockCacheGetter interface {
 }
 
 type diskBlockCacheSetter interface {
-	SetDiskBlockCache(DiskBlockCache)
+	ResetDiskBlockCache(ctx context.Context) error
```

Comment:
I left it there to give us the option of restarting it, but I can remove it if you'd prefer.